manim/config/config.py

```diff
@@ -124,8 +124,14 @@ def _parse_config(config_parser, args):
     config["left_side"] = config["frame_x_radius"] * constants.LEFT
     config["right_side"] = config["frame_x_radius"] * constants.RIGHT
 
-    # Handle the --tex_template flag.  Note we accept None if the flag is absent
-    tex_fn = os.path.expanduser(args.tex_template) if args.tex_template else None
+    # Handle the --tex_template flag, if the flag is absent read it from the config or accept None.
+    if args.tex_template:
+        tex_fn = os.path.expanduser(args.tex_template)
+    else:
+        try:
+            tex_fn = default["tex_template"]
+        except KeyError:
+            tex_fn = None
```

Comment:
I'm really sorry, but I can never remember correctly who is responsible for the tex_template stuff. @cobordism ? @PhilippImhof ?

Whoever knows the answer: what would be an acceptable default value for `config["tex_template"]` if none is passed via CLI? Is None okay?

Reply:
I will answer and see wheter @cobordism has a better idea. IMHO `None` is a good default, because currently we already set it to `None` to indicate that the default template should be used.

https://github.com/ManimCommunity/manim/blob/16e18b26e5e2e230aa46c85fc04fdebf9f2fe1bf/manim/config/config.py#L127-L140

Reply:
the default template is `TexTemplate()`

Reply:
Right. That's what happens, if `tex_fn` is set to `None` (cf. L139). Or am I missing something?

Reply:
Yes, that's correct @PhilippImhof. If neither the command-line argument nor the config value is set it defaults to its old behaviour which is to set `tex_fn` to `None`.

Reply:
Thanks all. Yes currently the default is None, but I'm asking if there's something better than that. Apparently TexTemplate() is the way to go.

@jvdoorn, in order to fix #450, this PR would need to change default.cfg as well.

Reply:
Sorry for the weekend pings folks!

Reply:
> Thanks all. Yes currently the default is None, but I'm asking if there's something better than that. Apparently TexTemplate() is the way to go.
> 
> @jvdoorn, in order to fix #450, this PR would need to change default.cfg as well.

Done!